docs/content/v2024.2/reference/configuration/yb-tserver.md

```diff
@@ -1820,6 +1820,14 @@ See [transaction isolation levels](../../../architecture/transactions/isolation-
 
 Default: `read committed`
 
+##### yb_skip_redundant_update_ops
+
+PostgreSQL parameter to enable skipping updates to columns that are part of secondary indexes and constraint checks when the column values remain unchanged.
```

Comment:
@sushantrmishra "PostgreSQL parameter" implies a PG equvalent but I don't see such in PG docs

Reply:
@ddhodge This is a YB specific feature which does not exist in PG.